tradeexecutor/state/visualisation.py

```diff
@@ -56,12 +57,15 @@ class Plot:
     #: we use UNIX timestamp here to keep our state easily serialisable.
     points: Dict[int, float] = field(default_factory=dict)
 
+    #: Is the line horizontal-vertical. Used for stop loss line. See https://plotly.com/python/line-charts/?_ga=2.83222870.1162358725.1672302619-1029023258.1667666588#interpolation-with-line-plots
```

Comment:
Can we make this enum `PlotShape` where the options are `linear`, `horizontal_vertical` because we have more new shapes like this coming soon?